### PR TITLE
ci: make wheel builds more robust

### PR DESCRIPTION
**Pull request by hsheth2**
## Checklist

- [ ] The PR conforms to DataHub's [Contributing Guideline](https://github.com/datahub-project/datahub/blob/master/docs/CONTRIBUTING.md) (particularly [Commit Message Format](https://github.com/datahub-project/datahub/blob/master/docs/CONTRIBUTING.md#commit-message-format))
- [ ] Links to related issues (if applicable)
- [ ] Tests for the changes have been added/updated (if applicable)
- [ ] Docs related to the changes have been added/updated (if applicable). If a new feature has been added a Usage Guide has been added for the same.
- [ ] For any breaking change/potential downtime/deprecation/big changes an entry has been made in [Updating DataHub](https://github.com/datahub-project/datahub/blob/master/docs/how/updating-datahub.md)

### CODE REVIEW

File: docs-website/sphinx/Makefile [hunk 1]
```diff
@@ -22,7 +22,7 @@ $(VENV_SENTINEL): requirements.txt
 	$(VENV_DIR)/bin/pip install -r requirements.txt
 	touch $(VENV_SENTINEL)
 
-.PHONY: help html doctest linkcheck clean serve md
+.PHONY: help html doctest linkcheck clean clean_all serve md
```
Comment (asikowitz) [inline docs-website/sphinx/Makefile:25]: Can remove `clean` from here since `clean_all` covers that

File: docs-website/sphinx/requirements.txt [hunk 2]
```diff
@@ -1,4 +1,4 @@
--e ../../metadata-ingestion[datahub-rest]
+-e ../../metadata-ingestion[datahub-rest,sql-parsing]
```
Comment (asikowitz) [inline docs-website/sphinx/requirements.txt:1]: What's this for?

**Review by asikowitz — APPROVED** [review]

File: docs-website/sphinx/Makefile (hunk 1 above)
Comment (hsheth2) [reply docs-website/sphinx/Makefile:25]: yeah - doesn't hurt to have it, but agree that it's redundant

File: docs-website/sphinx/requirements.txt (hunk 2 above)
Comment (hsheth2) [reply docs-website/sphinx/requirements.txt:1]: DataHubGraph has a lineage method that uses sqlglot, so this helps sphinx not complain

doesn't actually impact much though

File: docs-website/sphinx/Makefile (hunk 1 above)
Comment (hsheth2) [reply docs-website/sphinx/Makefile:25]: actually this is the `.PHONY` rule so it's required here

**Pull request merged**